src/pokemon/Pokemon.java

```diff
@@ -0,0 +1,54 @@
+package pokemon;
+
+public abstract class Pokemon {
+    private String name;
+    private int hp;
+    private int atk;
+    private double speed;
+
+    public Pokemon(String name, int hp, int atk, double speed) {
+        this.name = name;
+        this.hp = hp;
+        this.atk = atk;
+        this.speed = speed;
+    }
+
+    public void eats() {
+        System.out.println("No information on this pokemon in the Pokedex.");
+    }
+
+    public abstract void evolvesInto();
```

Comment:
Je override is in elke subklasse hetzelfde. Misschien had je beter een andere methode als abstract kunnen kiezen